etc/aqd.conf.defaults

```diff
@@ -345,6 +345,9 @@ gzip_output = false
 # Assume the webserver will decompress transparently as needed.
 # only used if gzip_output = true
 transparent_gzip = true
+template_extension = .tpl
```

Comment:
it would make sense to fix the 'Revert' commit to not delete this rather than re-add it in here.

Reply:
You mean that this line disappeared because of the revert commit? This is stange... That means that the original commit did some bad things! I'll double check and fix the revert commit in this case...